times_reader/transforms.py

```diff
@@ -1359,6 +1403,7 @@ def process_transform_insert(
             nrows = df.shape[0]
 
             # Standardize column names
+            # TODO: CommGrp is an alias of Other_Indexes. What happens if both are present?
```

Comment:
Really? But Part IV just says Other_Indexes is "To enter special dimensions that are required in certain attributes". That's so confusing...

Reply:
It is! Other_Indexes can hold a lot of dimensions, including commodity group, also at the same time, i.e. `~`-seperated.
I guess, if one included a ``CommGrp` header in an `FT` table, one would only specify commodity group there... Should test what happens if both headers are included (e.g. would one overwrite the other?).